src/main/scala/gpp/ui/forms/EnumViewSelect.scala

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gpp.ui.forms
+
+import scalajs.js.JSConverters._
+
+import cats.Show
+import cats.implicits._
+import gem.util.Enumerated
+import japgolly.scalajs.react._
+import japgolly.scalajs.react.vdom.html_<^._
+import react.common.ReactProps
+import react.semanticui.addons.select.Select
+import react.semanticui.modules.dropdown._
+import crystal.ViewOptF
+import crystal.react.implicits._
+import cats.effect.Effect
+import scalajs.js
+
+/**
+  * Produces a dropdown menu, similar to a combobox
+  */
+final case class EnumViewSelect[F[_], A](
+  label:       String,
+  value:       ViewOptF[F, A],
+  placeholder: js.UndefOr[String] = js.undefined,
+  disabled:    js.UndefOr[Boolean] = js.undefined
+)(implicit
+  val enum:    Enumerated[A],
+  val show:    Show[A],
+  val effect:  Effect[F]
+) extends ReactProps[EnumViewSelect[Any, Any]](EnumViewSelect.component)
+
+object EnumViewSelect {
+  type Props[F[_], A] = EnumViewSelect[F, A]
+
+  implicit protected def propsReuse[F[_], A]: Reusability[Props[F, A]] =
+    Reusability.by(p => (p.label, p.value.get.map(p.show.show), p.placeholder, p.disabled))
+
+  protected val component =
+    ScalaComponent
+      .builder[Props[Any, Any]]
+      .stateless
+      .render_P { p =>
+        implicit val show   = p.show
+        implicit val effect = p.effect
+
+        <.div(
+          ^.cls := "field",
```

Comment:
Should we support adding a custom class using e.g. the `Css` type?

Reply:
Yes, that's a good idea. Actually, we may want to support all the properties from `Select`. Also, I think this should be a ` Field`.